src/scvi/model/_totalvi.py

```diff
@@ -1215,6 +1215,12 @@ def setup_anndata(
         -------
         %(returns)s
         """
+        warnings.warn(
+            "TOTALVI is suppose to work with MuData. the use of anndata is "
+            "deprecated and will be remove in scvi-tools 1.4. Please use setup_mudata",
```

Comment:
I think for TOTALVI, we should deprecate it, but without announcing that it will be removed in 1.4. For MultiVI, it's a bad solution, while for TOTALVI, it's actually quite okay'ish to use obsm.